packages/mobile/src/identity/contactMapping.ts

```diff
@@ -182,9 +186,15 @@ export function* fetchAddressesAndValidateSaga({ e164Number }: FetchAddressesAnd
 
     const userAddress = yield select(currentAccountSelector)
     const secureSendPhoneNumberMapping = yield select(secureSendPhoneNumberMappingSelector)
+    const possibleAddresses = addresses || []
```

Comment:
Maybe we could have `getAddresses` return an empty array instead of defined another var for this here. Just because it's a bit confusing to have `oldAddresses`, `addresses`, and now `possibleAddresses` in the same function